feat: Beagle Configuration on entrypoint doc (#902)
ZupIT/beagle-docs: content/en/ios/customization/dependencies/network-layer.md

## Introduction

> Beagle makes all the web request from the **`networkClient`** dependency that is present in BeagleDependencies of the type `NetworkClientProtocol`, so it is possible to create your own network layer implementation to be used in the framework.

Reviewer: ```suggestion
Beagle makes all network requests through the **`networkClient`** dependency, which is present in the BeagleDependencies of type `NetworkClientProtocol`. This allows developers to create their own network layer implementation to be used by the framework.
```
